schedule/yast/autoyast_bcache.yaml

```diff
@@ -0,0 +1,71 @@
+---
+name: autoyast_bcache
+description: |
+  Test installation with AY profile for bcache with encrypted lvm.	
+  One contains bios boot partition, /boot and not formatted native linux partition.
+  Bcache uses not formatted native linux partition as backing device and second disk as a whole as caching device.
+vars:
+  NUMDISKS: 2
+  AUTOYAST_PREPARE_PROFILE: 1
+  AUTOYAST: autoyast_sle15/autoyast_bcache.xml
+schedule:
+  - autoyast/prepare_profile
+  - installation/isosize
+  - installation/bootloader_start
+  - autoyast/installation
+  - autoyast/console
+  - autoyast/login
+  - autoyast/clone
+  - autoyast/repos
+  - autoyast/logs
+  - autoyast/autoyast_reboot
+  - installation/grub_test
+  - installation/first_boot
+  - console/verify_bcache_partitioning
+  - autoyast/verify_cloned_profile
+test_data:
+  backingdev: vda4
+  profile:
+    partitioning:
+      - drive:
+          unique_key: device
+          device: /dev/vda
+          type: CT_DISK
+          partitions:
+            - partition:
```

Comment:
validator for cloned profile doesn't want to be smart if you just include a subset of partitions, AutoYaST generates the boot partition by default, so you are missing that one here so it will match 4.

Reply:
Dawm. thanks for it. I am stupid as i was looking to a wrong xml too.

Reply:
no worries, happy to help ;)